pkg/clicmd/prototype_use.go

```diff
@@ -84,34 +87,80 @@ ks prototype use single-port-deployment nginx-depl \
 `
 )
 
-func newGenerateCmd(a app.App) *cobra.Command {
-	cmd := newPrototypeUseCmd(a)
+func newGenerateCmd(fs afero.Fs) *cobra.Command {
+	cmd := newPrototypeUseCmd(fs)
 	cmd.Use = "generate <prototype-name> <component-name> [type] [parameter-flags]"
 
 	return cmd
 }
 
-func newPrototypeUseCmd(a app.App) *cobra.Command {
+func newPrototypeUseCmd(fs afero.Fs) *cobra.Command {
 	prototypeUseCmd := &cobra.Command{
-		Use:                "use <prototype-name> <componentName> [type] [parameter-flags]",
-		Short:              protoShortDesc["use"],
-		Long:               prototypeUseLong,
-		Example:            prototypeUseExample,
+		Use:     "use <prototype-name> <componentName> [type] [parameter-flags]",
+		Short:   protoShortDesc["use"],
+		Long:    prototypeUseLong,
+		Example: prototypeUseExample,
+		FParseErrWhitelist: cobra.FParseErrWhitelist{
+			UnknownFlags: true,
+		},
 		DisableFlagParsing: true,
-		RunE: func(cmd *cobra.Command, rawArgs []string) error {
-			if len(rawArgs) == 1 && (rawArgs[0] == "--help" || rawArgs[0] == "-h") {
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.DisableFlagParsing = false
```

Comment:
Is there a way to avoid mutating the command here? Perhaps we can create a copy first?

Reply:
What had bothered me was that a second execution would behave differently because of the mutation - largely academic given current application structure. Maybe just reset the flag the way you found it afterwards?

Reply:
resetting it after sounds good actually - thanks for the suggestion